lib/kubernetes/constants.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
Was this file copied from K8s source code?

Reply:
Oh, oops. Yeah, looks like I copied the header from the kubernetes src.